tiaas/logging.py

```diff
@@ -0,0 +1,34 @@
+"""Logging configuration."""
+
+LOGGING = {
+    'version': 1,
+    'disable_existing_loggers': False,
+    'formatters': {
+        'verbose': {
+            'format': '{levelname} | {asctime} | {module}: {message}',
+            'style': '{',
+        },
+    },
+    'handlers': {
+        'console': {
+            'level': 'INFO',
+            'class': 'logging.StreamHandler',
+            'formatter': 'verbose',
+        },
+        'file': {
+            'level': 'INFO',
+            'class': 'logging.handlers.RotatingFileHandler',
+            'filename': '/tmp/tiaas.log',
```

Comment:
I'd really rather we just log to stdout than a file, this is essentially *always* deployed as a systemd unit, `journalctl -fu tiaas` works as a result (just like with other galaxy processes), and journalctl handled the rotation and cleanup of old logs automatically.

Reply:
Yep, agreed. I'll fix that.

Reply:
What do you think about optional logging to an email handler on error? I usually do that in production deployments so we can keep on top of bug as they pop up. Or do admins tend to monitor their syslogs with something like Sentry for this purpose?